packages/gatsby-theme-carbon/src/components/Aside/Aside.js

```diff
@@ -1,29 +1,15 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import classnames from 'classnames';
```

Comment:
Sometime we are importing as `classnames`, but most of the time we import as `cx`. Should we standardize on just using `cx` for consistency? Or does that not really matter? Also, *this is awesome*!

Reply:
Yeah we totally should, I'll double check this

Reply:
@sstrubberg went through and normalized everything towards cx. It might conflict with some of your stuff so we'll merge your PR first